ports/libsimpleservo/api/src/lib.rs

```diff
@@ -55,14 +58,14 @@ pub use servo::embedder_traits::EventLoopWaker;
 
 pub struct InitOptions {
     pub args: Vec<String>,
-    pub url: Option<String>,
     pub coordinates: Coordinates,
     pub density: f32,
     pub xr_discovery: Option<webxr::Discovery>,
     pub enable_subpixel_text_antialiasing: bool,
     pub gl_context_pointer: Option<*const c_void>,
     pub native_display_pointer: Option<*const c_void>,
     pub native_widget: *mut c_void,
+    pub prefs: Option<HashMap<String, PrefValue>>,
 }
```

Comment:
Now that we can pass preferences to the embedder, there's not need to also pass the URL, and the embedder can just set homepage pref.